ethcore/src/snapshot/account.rs

```diff
@@ -170,6 +179,12 @@ pub fn from_fat_rlp(
 		return Ok((ACC_EMPTY, None));
 	}
 
+	let use_short_version = match rlp.item_count()? {
+		5 => true,
```

Comment:
I think, that it's a risky logic, to rely on item count in rlp packet. May be to pass this parameter explicitly?

Reply:
I don't get what you mean here. Which parameter do you think we can pass explicitly?